src/app/components/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { textStyle } from './Typography/base';
+
+const margin = 3.0;
+
+// tslint:disable-next-line:variable-name
+const Card = styled.div`
```

Comment:
if this `Card` component is supposed to be generic can you try to make it work with the notification popup? if not then you might as well just put these components in the ErrorModal component

Reply:
I took a look at doing so, but the notification popup is quite a bit different. it changes the layout to be a top bar depending on mobile size.

Reply:
so can we put these components in the ErrorModal component then?